pkgs/applications/science/misc/colmap/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, lib, fetchFromGitHub, cmake, boost, ceres-solver, eigen,
+  freeimage, glog, libGLU, glew, qt5,
+  cudaSupport ? false, cudatoolkit ? null }:
+
+assert !cudaSupport || cudatoolkit != null;
+
+let boost_static = boost.override {enableStatic = true; };
+in 
+stdenv.mkDerivation rec {
+  version = "3.5";
+  name = "colmap-" + version;
+  src = fetchFromGitHub {
+     owner = "colmap";
+     repo = "colmap";
+     rev = version;
+     sha256 = "1vnb62p0y2bnga173wmjs0lnyqdjikv0fkcxjzxm8187khk2lly8";
+  };
+  
+  buildInputs = [ boost_static ceres-solver eigen freeimage
```

Comment:
There must be a nicer way to format this list

Reply:
I would personally put a newline after `[` and before `]` but I get this is personal preference